### ISSUE TITLE
Ctrl-C copied recipe not shown in correct folder until after restart

### ISSUE THREAD

**Issue by Brew1984** [Brewtarget/brewtarget#909]
Windows, 4.0.12: After copying a recipe nothing happens. Only after restarting 4.0.12 it appears.

**Comment by Brew1984** [comment]
Only if I use the right click option. When using the menu option it's ok.

**Comment by matty0ung** [comment]
Ah, I see it - thanks.  On my install, right-click copy works, Ctrl-C copy doesn't show up in the display.  I'll take a look at what the differences are.

**Comment by matty0ung** [comment]
OK, checked this on Linux too and it's the same problem.  Basically, when you right-click and select copy, `TreeView::copySelected` is called, which does all the right things.  When you press Ctrl-C (or use the main menu), `MainWindow::copyRecipe()` is invoked instead, which does almost all the right things, but doesn't update the display to show the copied recipe in its correct folder.  (The recipe is actually displayed, but it's usually right at the bottom of the list, so probably not on-screen unless you have a very large monitor!)

The fix will be to make `MainWindow::copyRecipe()` call `TreeView::copySelected` rather than have its own logic.

**Comment by matty0ung** [comment]
Fix is included in https://github.com/Brewtarget/brewtarget/pull/908 and should be in the next release.

**Comment by matty0ung** [comment]
Fix included in [4.0.13](https://github.com/Brewtarget/brewtarget/releases/tag/v4.0.13). As ever, please don't hesitate to reopen this issue if you're still seeing this problem (or to raise a new issue if you hit any other problems).
